.github/workflows/flink-ci.yml

```diff
@@ -20,25 +20,8 @@
 name: "Flink CI"
 on:
   push:
-    paths-ignore:
-    - '.github/workflows/python-ci.yml'
-    - '.github/workflows/spark-ci.yml'
-    - '.github/workflows/hive-ci.yml'
-    - '.github/workflows/cancel-duplicate-workflow-runs.yml'
-    - '.gitignore'
-    - 'dev/**'
-    - 'mr/**'
-    - 'hive3/**'
-    - 'hive3-orc-bundle/**'
-    - 'hive-runtime/**'
-    - 'spark/**'
-    - 'pig/**'
-    - 'python/**'
-    - 'python_legacy/**'
-    - 'docs/**'
-    - '.gitattributes'
-    - 'README.md'
-    - 'CONTRIBUTING.md'
+    branches:
```

Comment:
Adding this should work for tags, but I that's not necessary when CI's run on release branches as well and the tag is on a release branch.
```
  tags:
  - 'apache-iceberg-*'
```

Reply:
Oh good catch. I just added a tag of `**`. I can update it to the tag you suggested if you'd like.